packages/runtime/container-runtime/src/runningSummarizer.ts

```diff
@@ -263,15 +264,28 @@ export class RunningSummarizer implements IDisposable {
                 { refreshLatestAck: true, fullTree: false, delayMinutes: 2 },
                 { refreshLatestAck: true, fullTree: true, delayMinutes: 10 },
             ];
+            let overrideDelaySeconds: number | undefined;
+            let retryAttempt = 0;
             for (let i = 0; i < attempts.length; i++) {
                 const { delayMinutes = 0, ...options } = attempts[i];
-                if (delayMinutes > 0) {
-                    await new Promise((resolve) => setTimeout(resolve, delayMinutes * 1000 * 60));
+                const delaySeconds = overrideDelaySeconds ?? (delayMinutes * 60);
```

Comment:
Consider using seconds across the board. We have seen cases (not in summarizer) where mix of different things caused bugs, hidden for very long time until they would explode.

Reply:
I was trying to address this potential issue by always including the units in the variable name. But I will go ahead and change it just in case.